wremnants/syst_tools.py

```diff
@@ -218,11 +219,19 @@ def define_mass_weights(df, proc):
 
     return df
 
-def add_massweights_hist(results, df, axes, cols, base_name="nominal", proc=""):
+def add_massweights_hist(results, df, axes, cols, base_name="nominal", proc="", addhelicity=False):
     name = Datagroups.histName(base_name, syst="massWeight"+(proc[0] if len(proc) else proc))
-    massWeight = df.HistoBoost(name, axes, [*cols, "massWeight_tensor_wnom"], 
-                    tensor_axes=[hist.axis.StrCategory(massWeightNames(proc=proc), name="massShift")], 
```

Comment:
The problem could be here, you deleted this line, but setupCombineWmass requires the new names

Reply:
This is something I am also modifying in PR #257, but for now you need to ensure you use the name with the boson in it